src/meta/src/hummock/compaction_scheduler.rs

```diff
@@ -104,6 +106,8 @@ where
                     match compaction_group {
                         Some(compaction_group) => compaction_group,
                         None => {
+                            // There are no new compaction groups, lets wait for a bit.
+                            tokio::time::sleep(Duration::from_millis(200)).await;
```

Comment:
this is not necessary...anw, there is no need to break 'compaction_trigger, there is only one loop so break is fine.